lib/active_storage_validations/matchers/attached_validator_matcher.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require_relative 'concerns/active_storageable.rb'
+require_relative 'concerns/allow_blankable.rb'
```

Comment:
No need for this concern here, the `allow_blank: true` option is not allowed on this validator/matcher. It would be contradictory to allow blank AND require attachment.